benchmark/Streamly/Benchmark/Data/Stream/StreamK.hs

```diff
@@ -40,12 +40,87 @@ import Streamly.Benchmark.Common
 import Test.Inspection
 #endif
 
+{-
+
+Benchmarks that need to be added
+
+-- repeat
+-- repeatM
+-- replicate
+-- replicateM
+-- iterate
+-- iterateM
+-- fromList
+
+-- bindWith
+-- concatPairsWith
+-- apWith
+-- apSerial
+-- apSerialDiscardFst
+-- apSerialDiscardSnd
+
+-- the
+-- serial
+-- consMStream
+-- withLocal
+-- mfix
+
+-- elem
+-- notElem
+-- all
+-- any
+-- minimum
+-- minimumBy
+-- maximum
+-- maximumBy
+-- findIndices
+-- lookup
+-- findM
+-- find
+-- (!!)
+
+-- foldlM'
+-- foldlT
+-- foldlx'
+-- foldlMx'
+-- fold
+
+-- sequence
+
+-- foldrSM
+-- buildS
+-- buildM
+-- augmentS
+-- augmentSM
+-- foldr
+-- foldr1
+-- foldrM
+-- foldrT
+
+-- intersperseM
+-- insertBy
+-- deleteBy
+-- reverse
+-- mapMaybe
+
+-- zipWithM
+-- mergeBy
+-- mergeByM
+
+-- toStreamK (Probably can be skipped)
+-- hoist
+
+-- scanlx'
+
+-}
```

Comment:
Probably do not add all of them as it may take a lot of time, select a few important ones and add those. Especially those which are actually being used in Streamly.Prelude etc. A lot of these are not actually used.